SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -264,7 +264,7 @@ void GenerateSchemaFromClasses(const TArray<TSharedPtr<FUnrealType>>& TypeInfos,
 							   FComponentIdGenerator& IdGenerator)
 {
 	// Generate the actual schema.
-	FScopedSlowTask Progress((float)TypeInfos.Num(), LOCTEXT("GenerateSchemaFromClasses", "Generating Schema..."));
+	FScopedSlowTask Progress((float)TypeInfos.Num(), LOCTEXT("GenerateSchemaFromClasses", "Generating schema..."));
```

Comment:
There was inconsistency with upper/lowercase here so I just transitioned them all to lower

Reply:
Don't know if you want to do a drive by and change to a static case. Up to you.